src/main/java/study/step2/domain/Lotto.java

```diff
@@ -1,28 +1,63 @@
 package study.step2.domain;
 
+import static study.step2.utils.MessageUtil.LOTTO_NUMBER_DUPLICATED;
+import static study.step2.utils.MessageUtil.VALIDATOR_NUMBER_MESSAGE;
+import static study.step2.validator.Validator.NUMBER_OF_LOTTO_NUMBER;
+
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.Objects;
+import java.util.Set;
 
 public class Lotto {
 
-  private LottoNumber lottoNumber;
+  private Set<LottoNumber> lottoNumbers = new HashSet<>();
 
   public Lotto() {
   }
 
-  public Lotto(LottoNumber lottoNumber) {
-    this.lottoNumber = lottoNumber;
+  public Lotto(Set<LottoNumber> lottoNumbers) {
+    this.lottoNumbers = lottoNumbers;
+  }
+
+  public Lotto(String inputText) {
+    Arrays.stream(inputText.split(", "))
+        .forEach(inputNumber -> lottoNumbers.add(LottoMachine.getLottoNumberMap().get(Integer.parseInt(inputNumber))));
+  }
```

Comment:
```suggestion
  public Lotto(String inputText) {
    lottoNumbers = Arrays.stream(inputText.split(","))
            .map(String::trim)
            .map(LottoNumber::new)
            .collect(Collectors.toSet());
  }
```
이런식으로 표현할 수 있도록 만들어보면 어떨까요?

Reply:
기존 코드에는 LottoMachine에 1~45까지 숫자를 미리 만들어놨었는데요. 제안 해주신 부분들을 보면 또 다시 새로운 객체를 생성하게 되는건 아닌가요?

Reply:
그 부분을 LottoMachine이 아니라 LottoNumber에게 위임을 하면 되지 않을까요?